src/component/HomePage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function LandingPage() {
+  return (
+    <div className="wrapper">
+      <h1>Welcome to Math Magician</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        Quidem laudantium ad voluptatibus sit ab corrupti perferendis odio,
+        minus excepturi animi laborum corporis deserunt nesciunt non doloribus
+        quos possimus mollitia ea?
+      </p>
+    </div>
+  );
+}
+
```

Comment:
Nice job styling your project😉. Let's try to make sure our style structure sticks with the wireframe as this is a requirement in this milestone😉.

<img width="811" alt="image" src="https://github.com/abdozayan12/Math-Magicians/assets/64643995/d56129ba-2d3f-4284-b254-56695079bda6">

Take note of the highlighted elements and ensure they are positioned rightly in your project. also, take note of the spacings (margin and padding) and make the necessary change😉.

yours
<img width="908" alt="image" src="https://github.com/abdozayan12/Math-Magicians/assets/64643995/0e3bab72-0173-4a6f-9d4c-7830e9874fa7">

<img width="932" alt="image" src="https://github.com/abdozayan12/Math-Magicians/assets/64643995/f12656f5-4069-44b1-9f22-488b6bf8cdd2">